ext/EffectsMixedModelsExt.jl

```diff
@@ -29,4 +29,84 @@ function Effects.effects!(reference_grid::DataFrame, model::MixedModel;
     return reference_grid
 end
 
+"""
+    effects!(reference_grid::DataFrame, model::MixedModel, boot::MixedModelBootstrap;
+             eff_col=nothing, err_col=:err, typical=mean, invlink=identity,
+             lower_col=:lower, upper_col=:upper, level=nothing)
+
+Use the results of a bootstrap to compute empirical error estimates
+(instead of using the variance-covariance matrix).
+
+!!! warning
+    This method is experimental and may change in its defaults or
+    disappear entirely in a future release!
+"""
+function Effects.effects!(reference_grid::DataFrame, model::MixedModel,
+                          boot::MixedModelBootstrap;
+                          eff_col=nothing, err_col=:err, typical=mean, invlink=identity,
+                          lower_col=:lower, upper_col=:upper,
+                          level=nothing)
+    # right now this is written for a RegressionModel and implicitly assumes
+    # the existence of an appropriate formula method
+    form = formula(model)
+    form_typical = typify(reference_grid, form, modelmatrix(model); typical=typical)
+    # we don't need to worry about pivoting / fixef rank deficiency here
+    # because the default -0.0 placeholder coefs still yield the correct predictions
+    # it's the NaNs in the vcov that create problems, but this method sidesteps
+    # the vcov computation, so we're all good!
+    X = modelcols(form_typical, reference_grid)
+    eff = X * coef(model)
+    bootdf = unstack(DataFrame(boot.β), :coefname, :β)
+    β = Vector{Float64}(undef, ncol(bootdf) - 1)
+    errrep = mapreduce(vcat, 1:nrow(bootdf)) do iter
```

Comment:
The name `errorp` is a little confusing. A possible alternative:

```suggestion
    boot_eff = mapreduce(vcat, 1:nrow(bootdf)) do iter
```

Reply:
https://github.com/beacon-biosignals/Effects.jl/pull/75/commits/37b9c6aaa52d97544409d7c7eaf40ef380739a8c